src/ui/Pricing/Comparison/index.svelte

```diff
@@ -42,6 +48,7 @@
       {/if}
     </div>
   </Table>
+  <FeaturesToggle bind:active={isShowLess} onClick={showMoreFeatures} />
```

Comment:
1. Name is misleading and vaguely represent what this component does. Maybe, we can name it something like `<MoreFeaturesButton />`?
2. Use `bind` directive -> `bind:isShowingMore`